rundeckapp/src/main/groovy/com/dtolabs/rundeck/server/plugins/RundeckPluginRegistry.groovy

```diff
@@ -551,7 +551,6 @@ class RundeckPluginRegistry implements ApplicationContextAware, PluginRegistry,
                 if (bean instanceof PluginResourceLoader) {
                     return bean
                 }
-                log.warn("No resource loader for bean plugin: ${provider}. Provided bean: ${beanName}")
```

Comment:
Maybe we could change it to debug or trace instead of just deleting it? I'm not a fan of deleting log messages.

Reply:
it could be debug but i don't think it's very useful. most bean plugins don't implement that interface, so it's going to be just a lot of noise.

Reply:
yea i think you're right.